app.js

```diff
@@ -53,9 +53,18 @@ window.AppData = new Proxy(defaultData, {
 //   });
 // }, 8000);
 
+function padNum(num) {
+  // input '1' => output '01'
+  return ('00' + num).substr(-2,2);
+}
+
 function formatTime(c, d) {
   // example:   0:35 / 2:33
-  return ((c / 60).toFixed(2) + ' / ' + (d / 60).toFixed(2)).replace('.', ':');
+  const cMin = Math.floor(c / 60);
+  const cSec = Math.floor(c - cMin * 60);
+  const dMin = Math.floor(d / 60);
+  const dSec = Math.floor(d - dMin * 60);
+  return `${cMin}:${padNum(cSec)} / ${dMin}:${padNum(dSec)}`;
```

Comment:
It's a bit edge case, but YouTube does have a few full length movies, or other tech-like videos that exceed 60 minutes.
There may be better ways to format timestamps, but possibly not worth the complexity of creating a new Date object and then trying to use Date and then handle some grosser logic:

```js
function formatTime(seconds) {
  const timestamp = new Date(seconds * 1000);
  const utcTimestamp = new Date(
    timestamp.getUTCFullYear(),
    timestamp.getUTCMonth(),
    timestamp.getUTCDate(),
    timestamp.getUTCHours(),
    timestamp.getUTCMinutes(),
    timestamp.getUTCSeconds()
  );
  return utcTimestamp.toLocaleTimeString('en-US', {
    hour12: false,
    hour: '2-digit',
    minute: '2-digit',
    second: '2-digit'
  }).replace(/^0+:?/, '');
}

console.log(formatTime(7384)); // "2:03:04"
console.log(formatTime(654));  // "10:54"
console.log(formatTime(99));   // "01:39"
console.log(formatTime(9));    // "00:09"
```